README.md

```diff
@@ -249,6 +249,7 @@ We are looking for maintainers to add more parsers and to write query files for
 - [x] [markdown](https://github.com/MDeiml/tree-sitter-markdown) (experimental, maintained by @MDeiml)
 - [x] [markdown_inline](https://github.com/MDeiml/tree-sitter-markdown) (experimental, maintained by @MDeiml)
 - [x] [menhir](https://github.com/Kerl13/tree-sitter-menhir) (maintained by @Kerl13)
+- [ ] [mermaid](https://github.com/monaqa/tree-sitter-mermaid)
```

Comment:
```suggestion
- [ ] [mermaid](https://github.com/monaqa/tree-sitter-mermaid) (experimental)
```

Reply:
I'm assuming the script will change it back. Should I fix it here?

Reply:
If you rely on the script, you don't need to add anything to the README at all ;)

(I do appreciate adding that stuff in the original PR to avoid having to deal with automated followup PRs.)

Reply:
I did rely on the script and that's what it added to the README.

Reply:
Oh, I see, the script only adds this if there is a maintainer listed. Thanks for fixing the script -- if it's not too much work, could you leave that fix as a separate commit?